controllers/consumerCouponController.js

```diff
@@ -0,0 +1,109 @@
+const ConsumerCoupon = require("../models/ConsumerCoupon");
+
+const getAllConsumerCoupons = async (req, res) => {
+  try {
+    const consumerId = req.consumerId;
+
+    if (!consumerId) {
+      return res.status(404).json({ error: "Consumer not found" });
+    }
```

Comment:
"Consumer not found. Please log in."

Cause the consumer has to login to get the req.consumerId

Reply:
Missing, Please log in message.